content/_data/events/scale15x.adoc

```diff
@@ -0,0 +1,6 @@
+---
+title: "Scale 15x"
+date: "2017-03-02 thru 03-05T09:00:00"
```

Comment:
That data field is parsed as an actual date, and therefore the site will not generate without it. I suggest adding a file for each day (really on Saturday and Sunday matter)